scripts/dockerrun.sh

```diff
@@ -138,10 +144,7 @@ docker run $INTERACTIVE -t --rm --sig-proxy=true \
     -e RELEASESUFFIX \
     -e COREFXAZURECONTAINER \
     -e AZUREACCOUNTNAME \
-    -e AZUREACCESSTOKEN \
-    -e VSOPASSWORD \
     -e RELEASETOOLSGITURL \
     -e CORESETUPBLOBROOTURL \
-    -e CORESETUPBLOBACCESSTOKEN \
```

Comment:
Isn't this variable still being used in Prepare.targets?  If you remove this, it won't be passed through to the docker container.

Reply:
@eerhardt 
All access tokens/passwords are VSO secrets now. They cannot be passed thru via the environment  - they must be explicitly set on a command line; typically as a MSBuild property.